packages/react-flicking/src/react-flicking/index.tsx

```diff
@@ -77,13 +77,15 @@ export default class Flicking extends React.Component<IFlickingProps> {
     const panel = flicking._conf.panel;
     const horizontal = this.props.horizontal;
     const className = `${this.props.prefix}-panel`;
-    panel.$list = Array.prototype.slice.call(flicking.$container.children);
+    panel.$list = [].slice.call(flicking.$container.children);
     panel.count = panel.$list.length;
-    panel.origCount = panel.count;
+    panel.origCount = [].concat(this.props.children).length;
```

Comment:
@daybrush Could it be panel.$list different from `this.props.children`?

Reply:
@jongmoon 
$list is a panel list with clones.
origCount is the number of original panels except for clones.

Reply:
@daybrush I got it!